sql/mysqld.cc

```diff
@@ -6194,6 +6194,19 @@ int mysqld_main(int argc, char **argv)
   if (!server_id_supplied)
     LogErr(INFORMATION_LEVEL, ER_WARN_NO_SERVERID_SPECIFIED);
 
+  /* Server generates uuid after innodb is initialized. But during
+  initialization, if tablespaces like system, redo, temporary are encrypted,
+  they are initialized with "empty" UUID. Now UUID is available, fix the
+  empty UUID of such tablespaces now */
+  if (innodb_hton != NULL && innodb_hton->fix_tablespaces_empty_uuid) {
```

Comment:
s/NULL/nullptr. Also either both predicates should be != nullptr either if (innodb_hton && innodb_hton->fix_tablespaces_empty_uuid)